client/components/tooltip/style.scss

```diff
@@ -43,6 +43,8 @@
 			transform: translate( -50%, 22px );
 			border: solid 15px transparent;
 			border-top-color: $gray-900;
+			// Ensure the tooltip arrow does not obscure the mouse target element.
+			pointer-events: none;
 		}
 	}
```

Comment:
Using a lighter blue color from the `@wordpress/base-styles` library.

https://github.com/WordPress/gutenberg/blob/d0fa04737ffda96b03bade9d724bc7fd842482d4/packages/base-styles/_colors.native.scss#LL33C23-L33C23